src/index.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-use-before-define */
+import './styles/main.scss';
+
+const clear = document.querySelector('.clear-btn');
+const list = document.querySelector('.todo-list');
+const input = document.getElementById('add-input');
+const form = document.getElementById('todoform');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+const red = 'linethrough';
+
+// first render
+renderTodos();
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  saveTodo();
+  localStorage.setItem('todos', JSON.stringify(todos));
+});
+
+function saveTodo() {
+  const todoValue = input.value;
+
+  todos.push({
+    value: todoValue,
+    completed: false,
+  });
+  input.value = '';
+  renderTodos();
+}
+
```

Comment:
- [ ] You add and remove are working but i am not able to edit an existing in any way. Kindly fix it :blush:

Reply:
@HAMMAS-SALEEM it works perfectly for me I will make a loom so you can see and maybe you can tell me what happened

Reply:
Hey @david63011
Sure dear you can tell me. That would be very nice of you :blush:

Reply:
here the video that shows the edit function working normally

https://www.loom.com/share/961bf9cdcbcf406e92b3e49d84cbce86